src/hotspot/share/oops/klass.hpp

```diff
@@ -194,6 +195,8 @@ class Klass : public Metadata {
   };
 #endif
 
+  KlassFlags  _misc_flags;
```

Comment:
On the line above (167) where _access_flags is defined, maybe leave a forwarding comment, something like:

```
// Some flags created by the JVM, not in the class file itself, are in _misc_flags below.
```

Reply:
Added this comment.  Thanks.